opm/parser/eclipse/OpmLog/OpmLog.hpp

```diff
@@ -24,6 +24,7 @@
 #include <cstdint>
 
 #include <opm/parser/eclipse/OpmLog/Logger.hpp>
+#include <opm/parser/eclipse/OpmLog/LogUtil.hpp>
```

Comment:
When I remove this line, build failed.

```
/private/miliu/project/opm/opm-parser/opm/parser/eclipse/OpmLog/OpmLog.cpp:42:33: error: ?Log? has not been declared
         const std::string msg = Log::prefixMessage(Log::MessageType::Info, message);
                                 ^
/private/miliu/project/opm/opm-parser/opm/parser/eclipse/OpmLog/OpmLog.cpp:42:52: error: ?Log? has not been declared
         const std::string msg = Log::prefixMessage(Log::MessageType::Info, message);
                                                    ^
/private/miliu/project/opm/opm-parser/opm/parser/eclipse/OpmLog/OpmLog.cpp:43:20: error: ?Log? has not been declared
         addMessage(Log::MessageType::Info, msg);
```

But Logger.hpp includes LogUtil.hpp, why build does not work?